.github/workflows/build.yml.in

```diff
@@ -137,9 +141,9 @@ jobs:
 
       - name: If using a lean-pr-release toolchain, uninstall
         run: |
-          if [[ `cat lean-toolchain` =~ ^leanprover/lean4-pr-releases:pr-release-[0-9]+$ ]]; then
-            echo "Uninstalling transient toolchain `cat lean-toolchain`"
-            elan toolchain uninstall `cat lean-toolchain`
+          if [[ $( cat lean-toolchain ) =~ ^leanprover/lean4-pr-releases:pr-release-[0-9]+$ ]]; then
```

Comment:
nit: I always assumed that `$(cat lean-toolchain)` with no spaces was more idiomatic